[Editor] Compilation warnings in Unity 6 Preview

**Describe the bug**
In edtior, I got two warnings due to deprecated methods in Unity 6 Preview (6000.0.14f1).

```
Packages\com.gustavopsantos.reflex\Editor\DebuggingWindow\UnityScriptingDefineSymbols.cs(29,36): warning CS0618: 'PlayerSettings.GetScriptingDefineSymbolsForGroup(BuildTargetGroup)' is obsolete: 'Use GetScriptingDefineSymbols(NamedBuildTarget buildTarget) instead'
```

```
Packages\com.gustavopsantos.reflex\Editor\DebuggingWindow\UnityScriptingDefineSymbols.cs(34,9): warning CS0618: 'PlayerSettings.SetScriptingDefineSymbolsForGroup(BuildTargetGroup, string)' is obsolete: 'Use SetScriptingDefineSymbols(NamedBuildTarget buildTarget, string defines) instead'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Follow the Greetings tutorial
2. Close the project, delete the "Library" folder
3. Open the project
4. See the two warnings

**Expected behavior**
No warnings.

**Desktop:**
 - OS: Windows 10
 - Version 8.5.1
 - Unity 6000.0.14f1

--- a/Assets/Reflex/Editor/DebuggingWindow/UnityScriptingDefineSymbols.cs
+++ b/Assets/Reflex/Editor/DebuggingWindow/UnityScriptingDefineSymbols.cs
@@ -1,5 +1,6 @@
 ﻿using System.Collections.Generic;
 using UnityEditor;
+using UnityEditor.Build;
 
 public static class UnityScriptingDefineSymbols
 {
@@ -26,12 +27,12 @@ public static class UnityScriptingDefineSymbols
     
     private static HashSet<string> GetSymbols(BuildTargetGroup platform)
     {
-        return new HashSet<string>(PlayerSettings.GetScriptingDefineSymbolsForGroup(platform).Split(';'));
+        return new HashSet<string>(PlayerSettings.GetScriptingDefineSymbols(NamedBuildTarget.FromBuildTargetGroup(platform)).Split(';'));
     }
     
     private static void SetSymbols(HashSet<string> symbols, BuildTargetGroup platform)
     {
-        PlayerSettings.SetScriptingDefineSymbolsForGroup(platform, string.Join(";", symbols));
+        PlayerSettings.SetScriptingDefineSymbols(NamedBuildTarget.FromBuildTargetGroup(platform), string.Join(";", symbols));
     }
 
     public static void Toggle(string symbol, BuildTargetGroup platform)

